config.js

```diff
@@ -4,7 +4,14 @@
 
 'use strict';
 
+const env = process.env.NODE_ENV || 'production';
+const localhost = () => `${process.env.LISTEN_HOST || '0.0.0.0'}:${process.env.PORT || 8080}`;
+const useMocks = env === 'production' || process.env.USE_MOCKS !== 'false';
```

Comment:
This condition seems backwards to me. We definitely *don't* want mocks for production. I'd set this condition to `process.env.USE_MOCKS ? process.env.USE_MOCKS === 'true' : env !== 'production'`

That way if the `USE_MOCKS` env var is set, then it is _always_ respected, but if it is not set then mocks are used only for non-production by default.

Reply:
I got my logic backwards.